smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -266,14 +266,14 @@ contract MixinKeys is
   * Returns true if _keyManager is the manager of the key
   * identified by _tokenId
    */
-  function isKeyManager(
+  function _isKeyManager(
     uint _tokenId,
     address _keyManager
   ) internal view
```

Comment:
Should we make this public instead?

Reply:
I don't know. We already have the public mapping `keyManagerOf` available. This seems like something more needed internally maybe?

Reply:
We can always make this public if the need arises. Will merge as-is for now.